packages/mobile/src/sentry/Sentry.ts

```diff
@@ -13,7 +13,11 @@ export async function installSentry() {
     Logger.info(TAG, 'installSentry', 'Sentry URL not found, skiping instalation')
     return
   }
-  Sentry.init({ dsn: SENTRY_URL, environment: DeviceInfo.getBundleId() })
+  Sentry.init({
+    dsn: SENTRY_URL,
+    environment: DeviceInfo.getBundleId(),
+    enableAutoSessionTracking: false,
```

Comment:
Reading up on this, I think we should enable it. https://docs.sentry.io/product/releases/health/
```suggestion
    enableAutoSessionTracking: true,
```